runtime/frequency/src/development_genesis.rs

```diff
@@ -0,0 +1,154 @@
+		},
+		session: SessionConfig {
+			keys: invulnerables
+				.into_iter()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                 // account id
+						acc,                         // validator id
+						template_session_keys(aura), // session keys
+					)
+				})
+				.collect(),
+		},
+		// no need to pass anything to aura, in fact it will panic if we do. Session will take care
+		// of this.
+		aura: Default::default(),
+		aura_ext: Default::default(),
+		#[cfg(any(not(feature = "frequency-no-relay"), feature = "frequency-lint-check"))]
+		parachain_system: Default::default(),
+		#[cfg(any(not(feature = "frequency"), feature = "frequency-lint-check"))]
+		sudo: SudoConfig {
+			// Assign network admin rights.
+			key: Some(get_account_id_from_seed::<sr25519::Public>("Alice")),
+		},
+		schemas: crate::pallet_schemas::GenesisConfig {
+			initial_schemas: load_genesis_schemas(),
+			..Default::default()
+		},
+		time_release: Default::default(),
+		democracy: Default::default(),
+		treasury: Default::default(),
+		council: CouncilConfig { phantom: Default::default(), members: council_members },
+		technical_committee: TechnicalCommitteeConfig {
+			phantom: Default::default(),
+			members: technical_committee_members,
+		},
+	};
+
+	serde_json::json!(&genesis)
+}
+
+pub fn development_endowed_accounts() -> Vec<AccountId> {
+	vec![
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		get_account_id_from_seed::<sr25519::Public>("Bob"),
+		get_account_id_from_seed::<sr25519::Public>("Charlie"),
+		get_account_id_from_seed::<sr25519::Public>("Dave"),
+		get_account_id_from_seed::<sr25519::Public>("Eve"),
+		get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+		get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+		common_runtime::constants::TREASURY_PALLET_ID.into_account_truncating(),
+	]
+}
+
+pub fn development_invulnerables() -> Vec<(AccountId, AuraId)> {
+	vec![
+		(
+			get_account_id_from_seed::<sr25519::Public>("Alice"),
+			get_collator_keys_from_seed("Alice"),
+		),
+		(get_account_id_from_seed::<sr25519::Public>("Bob"), get_collator_keys_from_seed("Bob")),
+	]
+}
+
+pub fn development_council_members() -> Vec<AccountId> {
+	vec![
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		get_account_id_from_seed::<sr25519::Public>("Charlie"),
+		get_account_id_from_seed::<sr25519::Public>("Eve"),
+	]
+}
+
+pub fn development_technical_committee_members() -> Vec<AccountId> {
+	vec![
+		get_account_id_from_seed::<sr25519::Public>("Bob"),
+		get_account_id_from_seed::<sr25519::Public>("Dave"),
+	]
+}
+
+/// Helper function to generate a crypto pair from seed
+#[allow(clippy::expect_used)]
```

Comment:
add comment // The panic from expect will not occur here because these input values are hardcoded.